applications/editor.lisp

```diff
@@ -1547,6 +1569,60 @@ If no such form is found, then return the CL-USER package."
 (defun beginning-of-top-level-form-command ()
   (beginning-of-top-level-form (current-buffer *editor*)))
 
+(defvar *isearch-string* (make-array 0 :fill-pointer t))
+(defvar *last-isearch-string* *isearch-string*)
```

Comment:
These defvars should be bound in editor-main and the array should be created with ":element-type 'character :adjustable t"